packages/front-generator/src/generators/react-native/app/index.ts

```diff
@@ -0,0 +1,86 @@
+import {BaseGenerator, readProjectModel} from '../../../common/base-generator';
```

Comment:
In this file there is code duplication with ReactTSAppGenerator described in react-typescript/app/index.ts. Refactoring needed, but may be in future issues.

Reply:
Created a separate issue #86 for that as there is also code duplication between different React and Polymer generators